packages/react-storage/src/components/StorageBrowser/displayText/libraries/en/uploadView.ts

```diff
@@ -79,6 +80,19 @@ export const DEFAULT_UPLOAD_VIEW_DISPLAY_TEXT: DefaultUploadViewDisplayText = {
 
     return { content: 'All files uploaded.', type };
   },
+  getFilesValidationMessage: ({ invalidFiles = [] } = {}) => {
+    const tooBigFileNames = invalidFiles
+      .filter(({ file }) => isFileTooBig(file))
+      .map(({ file }) => file.name)
+      .join(', ');
+    if (tooBigFileNames) {
+      return {
+        content: `Files larger that 160GB cannot be added to the upload queue: ${tooBigFileNames}`,
```

Comment:
my comment had a typo :(

```suggestion
        content: `Files larger than 160GB cannot be added to the upload queue: ${tooBigFileNames}`,
```